src/main/java/org/opensearch/knn/index/query/rescore/RescoreContext.java

```diff
@@ -18,7 +18,7 @@ public final class RescoreContext {
 
     public static final float DEFAULT_OVERSAMPLE_FACTOR = 1.0f;
     public static final float MAX_OVERSAMPLE_FACTOR = 100.0f;
-    public static final float MIN_OVERSAMPLE_FACTOR = 0.0f;
+    public static final float MIN_OVERSAMPLE_FACTOR = 1.0f;
```

Comment:
[Question] should we have a UT validating this?

Reply:
Yes 

1. https://github.com/opensearch-project/k-NN/blob/main/src/test/java/org/opensearch/knn/index/query/parser/RescoreValidationTests.java#L33
2. https://github.com/opensearch-project/k-NN/blob/main/src/test/java/org/opensearch/knn/index/query/KNNQueryBuilderInvalidParamsTests.java#L95